### PR TITLE
docs: mark skipTaskbar as unsupported on Linux

### LINKED ISSUE #33124: [Bug]: skipTaskbar: true ignored
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

17.1.0-1-x86_64

### What operating system are you using?

Other Linux

### Operating System Version

Kernel: 5.16.11-arch1-1

### What arch are you using?

x86_64

### Last Known Working Electron version

16.0.8-2-x86_64

### Expected Behavior

electron app not shown in the taskbar

### Actual Behavior

electron app is shown in the taskbar

### Testcase Gist URL

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by ckerr**
BREAKING CHANGE

#### Description of Change

Closes #33124.

`skipTaskbar` has no effect on Wayland. Currently Electron uses `_NET_WM_STATE_SKIP_TASKBAR` to tell the WM to hide an app from the taskbar, and this works fine on X11 but there's no equivalent mechanism in Wayland. There _are_ some workarounds available on different systems, e.g. GNOME and KWin, but they some unacceptable tradeoffs, e.g. `Window.is_skip_taskbar` requires unsafe mode.

tl;dr I don't see a way to support this feature on Linux anymore.

This PR updates the documentation to reflect that it's only supported on macOS / Windows and makes an announcement in "breaking changes" (it's broken already, but better to document it here for visibility).

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/main/docs/development/testing.md)
- [ ] relevant documentation is changed or added
- [ ] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: Marked `.skipTaskbar` as unsupported on Linux.

### CODE REVIEW

**Review by codebytere — APPROVED** [review]

**Review by zcbenz — APPROVED** [review]

**Review by dsanders11 — APPROVED** [review]

**Review by jkleinsc — APPROVED** [review]
Should we mark this as deprecated versus removing it

File: docs/breaking-changes.md [hunk 1]
```diff
@@ -27,9 +27,20 @@ preload scripts _do_ depend on Node, either refactor them to remove Node usage
...
+### Removed: `skipTaskbar` on Linux
+
+See `skipTaskbar` discussion in 19.0 below. This feature is not available on
+Wayland. Since most modern Linux desktops are transitioning to Wayland, this
+feature will be removed for Linux.
+
 ## Planned Breaking API Changes (19.0)
 
-*None (yet)*
+### Unsupported: `skipTaskbar` on Linux
```
Comment (nornagon) [inline docs/breaking-changes.md:38]: I missed this, but all breaking changes in this document should use one of the types described at the top of the document (API Changed / Behavior Changed / Default Changed / Deprecated / Removed).

What does `Unsupported` mean?

Comment (ckerr) [reply docs/breaking-changes.md:38]: Followup in https://github.com/electron/electron/pull/33479

**Comment by freddy-daniel** [comment]
Will this affect browserWindow.setSkipTaskbar?
